<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>open_window</class>
 <widget class="QMainWindow" name="open_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="openwdw_lbl_welcome">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>341</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Welcome to the Banking App</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="openwdw_btn_CSLogin">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>240</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>CUSTOMER LOGIN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openwdw_btn_ADLogin">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>240</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ADMIN LOGIN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openwdw_btn_exit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>360</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>EXIT</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>openwdw_btn_exit</sender>
   <signal>clicked()</signal>
   <receiver>open_window</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>274</x>
     <y>422</y>
    </hint>
    <hint type="destinationlabel">
     <x>393</x>
     <y>561</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
